index.html

```diff
@@ -30,14 +30,186 @@ <h1 class="info_box">
     </div>
     <div>
         <p class="contact_section"><a href="#">LET'S CONNECT</a></p>
+        
     </div>
+    <section>
         <ul>
-            <li><a href="#"><img src="cat.svg" width="1" height="15" alt="cat"></a></li>
-            <li><a href="#"><img src="bird.svg" width="1" height="15" alt="bird"></a></li>
-            <li><a href="#"><img src="m.svg" width="1" height="15" alt="M"></a></li>
-            <li><a href="#"><img src="n.png" width="1" height="15" alt="N"></a></li>
-            <li><a href="#"><img src="two.svg" width="1" height="15" alt="two"></a></li>
-         </ul>
-    <h3><a href="#">LET'S GET INTERACTIVE</a></h3>
+            <li><a href="#"><img src="/img/cat.svg" width="1" height="15" alt="cat"></a></li>
+            <li><a href="#"><img src="/img/bird.svg" width="1" height="15" alt="bird"></a></li>
+            <li><a href="#"><img src="/img/m.svg" width="1" height="15" alt="M"></a></li>
+            <li><a href="#"><img src="/img/n.png" width="1" height="15" alt="N"></a></li>
+            <li><a href="#"><img src="/img/two.svg" width="1" height="15" alt="two"></a></li>
+        </ul>
+        <h3><a href="#">LET'S GET INTERACTIVE</a></h3>
+    </section>
+
+        
+
+    <section class="section_works_container">
+       <div>
+            <img class="sec_in" src="/img/sndwk.jpg" alt="selfie">
+            <div class="tonic">
+                <h2>Tonic</h2>
+            </div>
+            <div>
+                <ul class="sec_h">
+                    <li><a href="#"><img src="/img/canopy.svg" alt="canopy"></a></li>
+                    <li><a href="#"><img src="/img/Count.svg" width="1" height="5" alt="count"></a></li>
+                    <li><a href="#"><img src="/img/bed.svg" height="22" alt="bed"></a></li>
+                    <li><a href="#"><img src="/img/cc.svg" width="1" height="5" alt="dot"></a></li>
+                    <li><a href="#"><img src="/img/time.svg" alt="date"></a></li>
```

Comment:
[OPTIONAL] instead of using image, write the text and design it using CSS to make your project more professional